nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutDatabaseRecord.java

```diff
@@ -1042,7 +1072,22 @@ private static String normalizeColumnName(final String colName, final boolean tr
         return colName == null ? null : (translateColumnNames ? colName.toUpperCase().replace("_", "") : colName);
     }
 
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    private static String convertMapRecord(Record record) {
```

Comment:
If we use a set of options for what to convert the record to (see my other comment), this method should be named accordingly for converting to a JSON string, as we would need other similar methods for other representations such as XML.